File: app/dbmodels/DeviceDriveItem.php
<?php

include "_commondeviceitems.php";

$table['model'] = 'DeviceDriveItem';

$table['fields']['interface_type_id'] = DBModels::type('integer');
$table['fields']['is_writer']        = DBModels::type('boolean');
$table['fields']['speed']            = DBModels::type('string');

$table['relationships']['interface_type'] = array(
                                                'type' => 'belongsTo',
                                                'item' => 'InterfaceType');

File: fusionglpi/database/device_drive_item.py
from fusionglpi import db

class DeviceDriveItem(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    entity_id = db.Column(db.Integer, db.ForeignKey('entity.id'))
    is_recursive = db.Column(db.Boolean, nullable=False, server_default="0")
    name = db.Column(db.String(255, None, True))
    manufacturer_id = db.Column(db.Integer, db.ForeignKey('manufacturer.id'))
    comment = db.Column(db.Text)
    interface_type_id = db.Column(db.Integer, db.ForeignKey('interface_type.id'))
    is_writer = db.Column(db.Boolean, nullable=False, server_default="0")
    speed = db.Column(db.String(255, None, True))
